v0/destinations/active_campaign/transform.js

```diff
@@ -467,9 +514,18 @@ const trackRequestHandler = async (message, category, destination) => {
         }
       }
     );
-  } catch (err) {}
+  } catch (err) {
+    throw new CustomError(
+      `Failed to retrieve all events (${err.response.statusText})`,
+      err.response.status || 400
+    );
+  }
+
   if (res.status !== 200)
-    throw new CustomError("Unable to fetch dest events", res.status || 400);
+    throw new CustomError(
+      `Unable to create dest event (${res.statusText})`,
+      res.status || 400
+    );
```

Comment:
The comments added above are applicable here too.

Reply:
Reverted to previous error message